### ISSUE TITLE
Have an uppercase severity formatter  [JIRA: RIAK-2489]

### ISSUE THREAD

**Issue by walrusVision** [erlang-lager/lager#347]
To make the severity level from larger match the way other programs log to syslog, can we please have a uppercase severity formatter? For example at the moment you will see this in the syslog logs:

> Apr  8 07:47:58 myhost other[28142]: INFO ...message
> Apr  8 07:47:58 myhost other[28142]: INFO ...message
> Apr  8 07:48:03 myhost erlang[23991]: info ...message
> Apr  8 07:48:03 myhost erlang[23991]: info ...message
> Apr  8 07:48:03 myhost other[28142]: INFO ...message
> Apr  8 07:48:03 myhost erlang[23991]: info ...message
> Apr  8 07:48:03 myhost other[28142]: INFO ...message

This actually makes it quite hard to skim read through.

So can we add a formatter like this:

``` erlang
output(severity_upper,Msg) ->
    string:to_upper(atom_to_list(lager_msg:severity(Msg)));
```

So that then with a config like this:

``` erlang
{handlers, [
  {lager_syslog_backend, [
    "erlang", user, debug, {lager_default_formatter, [severity_upper, " ", message]}
  ]}
]}
```

We can get logs like this

> Apr  8 07:47:58 myhost other[28142]: INFO ...message
> Apr  8 07:47:58 myhost other[28142]: INFO ...message
> Apr  8 07:48:03 myhost erlang[23991]: INFO ...message
> Apr  8 07:48:03 myhost erlang[23991]: INFO ...message
> Apr  8 07:48:03 myhost other[28142]: INFO ...message
> Apr  8 07:48:03 myhost erlang[23991]: INFO ...message
> Apr  8 07:48:03 myhost other[28142]: INFO ...message

**Comment by mrallen1** [comment]
Yeah, I can see that.  As far as code though, I'd much prefer something like this:

```erlang
output(severity_upper, Msg) ->
    uppercase_severity(lager_msg:severity(Msg)).

uppercase_severity(debug) -> "DEBUG";
uppercase_severity(info) -> "INFO";
...
uppercase_severity(emergency) -> "EMERGENCY".
```

**Comment by mrallen1** [comment]
Implemented in #372 so I'm closing this. Thanks!
